### DISCUSSION TITLE
Student Developer Pack approved but no benefit

### DISCUSSION THREAD

**Discussion by AhmedKahlaoui** [community/community#58944] [GitHub Education]
### Hi everyone,

Hello,

I have recieved confirmation from the [relevant application page](https://education.github.com/discount_requests/application) that my application has been approved:
![image](https://github.com/community/community/assets/123409443/448a441f-da03-4d9d-b09c-46e4eebf4b3a)
However, I have recieved no email from neither my primary, nor education email to state that I have recieved any benefits. Logging into my account shows no benefits whatsoever.

**Comment by the-officialjosh** [comment]
Wait it will be rolled out to you

**Comment by samus-aran** [comment]
Hey @AhmedKahlaoui  👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

**Comment by AshishKumar9532** [comment]
same happening with me too.
Account is approved for education pack but no benefits yet.
![Screenshot (182)](https://github.com/community/community/assets/99255937/7f6f102e-d5df-4a1a-b590-0f00cea22cf1)

**Reply by WaldperlachFabi** [reply]
Same here, it’s been 4 days…

**Reply by internettrashh** [reply]
did you get benifits?

**Reply by bugnyx** [reply]
Did you guys get your benefits yet? how long it did take for you guys?

**Reply by Meto423** [reply]
For me, it's approved one week ago but still waiting for benefits.

**Reply by Sortren** [reply]
same, been waitining for three days and nothing

**Reply by Meto423** [reply]
I got mine finally, so eventually you get the benefits but the application and waiting process is frustrating. But thanks anyway.

**Reply by Tostiffent** [reply]
> I got mine finally, so eventually you get the benefits but the application and waiting process is frustrating. But thanks anyway.

how long did it take

**Reply by Meto423** [reply]
> > I got mine finally, so eventually you get the benefits but the application and waiting process is frustrating. But thanks anyway.
> 
> how long did it take


It took approximately 2 weeks.

**Reply by InvictusNavarchus** [reply]
> > > I got mine finally, so eventually you get the benefits but the application and waiting process is frustrating. But thanks anyway.
> > 
> > 
> > how long did it take
> 
> It took approximately 2 weeks.

Holy moly, 2 weeks? I guess we just need to wait.  Thanks for the heads-up!
